src/sentry/api/endpoints/account_appearance_details.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import serializers
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.api.base import Endpoint
+from sentry.constants import LANGUAGES
+from sentry.models import UserOption
+
+
+def _get_timezone_choices():
+    results = []
+    for tz in pytz.common_timezones:
+        now = datetime.now(pytz.timezone(tz))
+        offset = now.strftime('%z')
+        results.append((int(offset), tz, '(UTC%s) %s' % (offset, tz)))
+    results.sort()
+
+    for i in range(len(results)):
+        results[i] = results[i][1:]
+    return results
+
+
+TIMEZONE_CHOICES = _get_timezone_choices()
+
+
+class AccountAppearanceSerializer(serializers.Serializer):
+    language = serializers.ChoiceField(choices=LANGUAGES, required=False)
+    stacktrace_order = serializers.ChoiceField(choices=(
+        ('-1', _('Default (let Sentry decide)')),
+        ('1', _('Most recent call last')),
+        ('2', _('Most recent call first')),
+    ), required=False)
+    timezone = serializers.ChoiceField(choices=TIMEZONE_CHOICES, required=False)
+    clock_24_hours = serializers.BooleanField(required=False)
+
+
+class AccountAppearanceDetailsEndpoint(Endpoint):
```

Comment:
What's a better name for this, it's not really a "details" endpoint

Reply:
what about just AccountAppearanceEndpoint?